JSONAPIMapper.podspec

```diff
@@ -0,0 +1,13 @@
+Pod::Spec.new do |spec|
+  spec.name         = 'JSONAPIMapper'
+  spec.summary      = 'An iOS library supports mapping json data'
```

Comment:
I think this sentence has a weird sound to it. Maybe. Currently it sounds like "An iOS library" is able to support "mapping json data"
```suggestion
  spec.summary      = 'An iOS library for mapping JSON data'
```
or better explanation would be "An iOS library supporting JSON data mapping to decodable"

Reply:
It also supports for macOS, so maybe `a Swift library for mapping JSON data` :D

Reply:
@blyscuit @suho updated guys 👍